modules/installation-aws-upi-requirements.adoc

```diff
@@ -0,0 +1,399 @@
+|`AWS::ElasticLoadBalancingV2::TargetGroup`
+|The target group for the Internal load balancer.
+
+|Internal listener
+|`AWS::ElasticLoadBalancingV2::Listener`
+|A listener on port 26443 for the internal load balancer.
+
+|Internal target group
+|`AWS::ElasticLoadBalancingV2::TargetGroup`
+|The target group for the Internal load balancer.
+
+|===
+
+.Security groups
+
+The control plane and worker machines require access to the following ports:
+
+[cols="2a,2a,2a,2a",options="header"]
+|===
+
+|Group
+|Type
+|IP Protocol
+|Port range
+
+
+.4+|MasterSecurityGroup
+.4+|`AWS::EC2::SecurityGroup`
+|`icmp`
+|`0`
+
+|`tcp`
+|`22`
+
+|`tcp`
+|`6443`
+
+|`tcp`
+|`22623`
+
+.2+|WorkerSecurityGroup
+.2+|`AWS::EC2::SecurityGroup`
+|`icmp`
+|`0`
+
+|`tcp`
+|`22`
+
+
+.2+|BootstrapSecurityGroup
+.2+|`AWS::EC2::SecurityGroup`
+
+|`tcp`
+|`22`
+
+|`tcp`
+|`19531`
+
+|===
+
+.Control plane ingress
+
+The control plane machines require the following ingress groups. Each ingress group is
+a `AWS::EC2::SecurityGroupIngress` resource.
+
+[cols="2a,5a,2a,2a",options="header"]
+|===
+
+|Ingress group
+|Description
+|IP protocol
+|Port range
+
+
+|`MasterIngressEtcd`
+|etcd
+|`tcp`
+|`2379`- `2380`
+
+|`MasterIngressVxlan`
+|Vxlan packets
+|`udp`
+|`4789`
+
+|`MasterIngressWorkerVxlan`
+|Vxlan packets
+|`udp`
+|`4789`
+
+|`MasterIngressInternal`
+|Internal cluster communication
+|`tcp`
+|`9000` - `9999`
+
+|`MasterIngressWorkerInternal`
+|Internal cluster communication
+|`tcp`
+|`9000` - `9999`
+
+|`MasterIngressKube`
+|Kubernetes kubelet, scheduler and controller manager
+|`tcp`
+|`10250` - `10252`
+
+|`MasterIngressWorkerKube`
+|Kubernetes kubelet, scheduler and controller manager
+|`tcp`
+|`10250` - `10252`
+
+|`MasterIngressIngressServices`
+|Kubernetes ingress services
+|`tcp`
+|`30000` - `32767`
+
+|`MasterIngressWorkerIngressServices`
+|Kubernetes ingress services
+|`tcp`
+|`30000` - `32767`
+
+|===
+
+
+.Worker ingress
+
+The worker machines require the following ingress groups. Each ingress group is
+a `AWS::EC2::SecurityGroupIngress` resource.
+
+[cols="2a,5a,2a,2a",options="header"]
+|===
+
+|Ingress group
+|Description
+|IP protocol
+|Port range
+
+
+|`WorkerIngressVxlan`
+|Vxlan packets
+|`udp`
+|`4789`
+
+|`WorkerIngressWorkerVxlan`
+|Vxlan packets
+|`udp`
+|`4789`
+
+|`WorkerIngressInternal`
+|Internal cluster communication
+|`tcp`
+|`9000` - `9999`
+
+|`WorkerIngressWorkerInternal`
+|Internal cluster communication
+|`tcp`
+|`9000` - `9999`
+
+|`WorkerIngressKube`
+|Kubernetes kubelet, scheduler and controller manager
+|`tcp`
+|`10250`
+
+|`WorkerIngressWorkerKube`
+|Kubernetes kubelet, scheduler and controller manager
+|`tcp`
+|`10250`
+
+|`WorkerIngressIngressServices`
+|Kubernetes ingress services
+|`tcp`
+|`30000` - `32767`
+
+|`WorkerIngressWorkerIngressServices`
+|Kubernetes ingress services
+|`tcp`
+|`30000` - `32767`
+
+|===
+
+
+.Roles and instance profiles.
+
+The machines require the following `AWS::IAM::Role` objects. You must also
```

Comment:
The permissions you have below are overly broad.  You can make them more specific (see the list [here][1]), but even that list is overly broad.  So regardless of what you do to the values below, you probably want to soften the "require" wording here.

[1]: https://github.com/openshift/installer/blob/27b24b8250293db120464095960f3b42cfaa94f5/pkg/asset/installconfig/aws/permissions.go

Reply:
I'll rewrite the into and link to the fine-grained permissions. I'm opening a new issue to update the table of permissions.